### ISSUE TITLE
Exceptions when using cycli with no authorization required

### ISSUE THREAD

**Issue by drfraser** [nicolewhite/cycli#54]
Hi,

I initially filed an issue with py2neo: 

https://github.com/technige/py2neo/issues/621 

and technige said the issue looks to actually be within cycli.  As stated, I was using Neo4j 3.1.1 and have the latest version of cycli off of pypi.

I have not updated Neo4j to 3.2 as there was some issue with updating the data in my database, haven't gotten around to seeing if they've fixed that yet. I forget the details, I got some vague sense that Neo4j had some issues around authorization at the time. But I haven't needed to revisit that project yet.

**Comment by nicolewhite** [comment]
Yeah I have unfortunately not tested cycli with Neo4j 3.1.1! I imagine I'll just have to update py2neo or something. I will take a look soon.
